packages/wonder-blocks-card/src/components/card.tsx

```diff
@@ -165,14 +185,17 @@ const Card = React.forwardRef(function Card(
         styles,
         labels,
         tag,
-        testId,
+        testId = "card",
```

Comment:
Could we leave `testId` uninitialized and only set the test id if it is provided? That way there aren't unnecessary data attributes in the DOM, and teams will have to explicitly pass in a test id if they want to use it! This is the behaviour I've observed in other components!

We can also only set the close button test id if the main `testId` prop is provided! What do you think?